packages/upgrade/test/__fixtures__/hooks/existingRqMix.trpc.tsx

```diff
@@ -0,0 +1,24 @@
+import { testReactResource } from '../../__helpers';
+import { initTRPC } from '@trpc/server';
+import { z } from 'zod';
+
+export const t = initTRPC.create();
+export const appRouter = t.router({
```

Comment:
You don't think we should just define this once and have it shared? Feels like unnecessary repetition